partials/application-accelerator/_configuration.hbs.md

```diff
@@ -206,17 +208,17 @@ Accelerator. The same properties are provided in the `accelerator` section of th
 accelerator:
   git_credentials:
     secret_name: git-credentials
-    username: GIT-USER-NAME
-    password: GIT-PASSWORD-OR-ACCESS-TOKEN
-    ca_file: CUSTOM-CA-CERT
+    username: GIT_USERNAME
```

Comment:
why use underscores?

Reply:
From looking at the K8s documentation, specifically their example YAMLs, they leverage underscores rather than dashes. At first, I was switching things over to using the form `$VARIABLE_NAME`, but wasn't too sure if it was too big of a jump to assume folks had `envsubst` available.

If this goes against the style for the TAP docs, can certainly revert.

Reply:
Will revert back to `-`